### PR TITLE
feat(fe): add box-title component

### LINKED ISSUE #157: [Feature] Common(Atom) -  BoxTitle
### 🎨 Component Image

image

image

### ✏️ Describe component

페이지 상단에 위치하는 타이틀 입니다!

이전 프로젝트의 [PageTop 컴포넌트](https://github.com/skkuding/skku-coding-platform/blob/master/frontend/src/pages/oj/components/PageTop.vue)와 동일합니다 😉

### ☁️ Props

None

`slot` 으로 커스텀 가능

### 🔎 Using Page

- Problem
- Contest

### 🏷 Required Component

None

### PR DESCRIPTION
**Pull request by goo314**
### Description

- BoxTitle 구현 close #157 
- slot name
  - title
  - subtitle

### Additional context

- 컴포넌트 이름을 제 마음대로 Box Title이라고 했는데 어떤가요?
---

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md)
- [x] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#pr-and-branch) and follow the [Commit Convention](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#commit-convention)
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

**Comment by SOLBI1028** [comment]
boxtitle 좋은것 같습니다~!~~! ㅎㅎ

**Comment by SOLBI1028** [comment]
story 작성할 때 variant 사용하는걸로 변경하면 좋을것 같아요! component code는 좋은 것 같습니다! 👍🏼
```
<Story :layout="{ type: 'grid' }">
    <Variant title="green">
      <Badge color="green">Badge</Badge>
    </Variant>
    ...
  </Story>
```
image

File: frontend/src/common/components/Atom/BoxTitle.story.vue [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    </BoxTitle>
+    <h2 class="mt-8 mb-2 text-lg">Another Example</h2>
+    <BoxTitle>
+      <template #title>
+        🏆 SKKU Coding Platform
+        <span class="text-[#FEB144]">Contests</span>
+      </template>
+      <template #subtitle>Compete with schoolmates & win the prizes!</template>
+    </BoxTitle>
+  </Story>
```
Comment (SOLBI1028) [inline frontend/src/common/components/Atom/BoxTitle.story.vue:20]: story 작성할 때 variant 사용하는걸로 변경하면 좋을것 같아요! component code는 좋은 것 같습니다! 👍🏼

```
<Story :layout="{ type: 'grid' }">
    <Variant title="green">
      <Badge color="green">Badge</Badge>
    </Variant>
    ...
</Story>
```
image

Comment (goo314) [reply frontend/src/common/components/Atom/BoxTitle.story.vue:25]: ㅎㅣ감사합니다 수정했습니다~! 😋

**Review by SOLBI1028 — APPROVED** [review]

**Pull request merged**